packages/planner/tests/package.json

```diff
@@ -18,12 +18,8 @@
   },
   "dependencies": {
     "jasmine-spec-reporter": "4.2.1",
-    "protractor": "5.2.2",
+    "protractor": "5.2.0",
     "typescript": "2.9.2"
   },
-  "peerDependencies": {},
-  "devDependencies": {
```

Comment:
Why have you removed the devDependencies it is giving error in protractor.conf.ts that `protractor.conf.ts can not read jasmine`

Reply:
I didn't get this error after removing @types/jasmine. @vikram-raj can you confirm this?

I removed these because even if these dependencies are installed, compiler is always looking for these packages in the parent node_modules and these should be in parent node_modules because we also use jasmine for our unit tests.

Reply:
@sahil143  @sanketpathak Tests are running without any error for me.

Reply:
It's failing not just for me but I've tested with other machines and its failing on those as well.